File: backend/src/main/java/org/icpclive/events/codeforces/api/results/CFStandings.java
package org.icpclive.events.codeforces.api.results;

import org.icpclive.events.codeforces.api.data.CFContest;
import org.icpclive.events.codeforces.api.data.CFProblem;
import org.icpclive.events.codeforces.api.data.CFRanklistRow;

import java.util.List;

/**
 * @author egor@egork.net
 */
public class CFStandings {
    public CFContest contest;
    public List<CFProblem> problems;
    public List<CFRanklistRow> rows;
}

File: cds/src/main/kotlin/org/icpclive/cds/codeforces/api/results/CFStandings.kt
package org.icpclive.cds.codeforces.api.results

import kotlinx.serialization.Serializable
import org.icpclive.cds.codeforces.api.data.CFContest
import org.icpclive.cds.codeforces.api.data.CFProblem
import org.icpclive.cds.codeforces.api.data.CFRankListRow

@Serializable
data class CFStandings(
    val contest: CFContest,
    val problems: List<CFProblem>,
    val rows: List<CFRankListRow>,
)